src/sentry/quotas/redis.py

```diff
@@ -67,6 +68,20 @@ def get_quotas(
 
         results = [*self.get_abuse_quotas(project.organization)]
 
+        # If the organization belongs to the killswitched list, we want to stop ingesting custom metrics.
+        if project.organization.id in (
```

Comment:
I opted to give this killswitch an explicit implementation here but in the future we could just rely on the abuse quota defined above.